src/main/java/org/opentripplanner/middleware/models/AbstractUser.java

```diff
@@ -26,13 +26,10 @@
     /** Whether a user is also a Data Tools user */
     public boolean isDataToolsUser;
     /**
-     * Set of permissions the user has.
+     * Set of permissions the user has (not persisted in Mongo).
```

Comment:
Please add FIXME and create bug/issue because we ultimately will want to persist these in Mongo.

Reply:
Noted in c424e4a and #41.